backend/src/main/java/com/bakdata/conquery/models/messages/network/specific/AddWorker.java

```diff
@@ -31,18 +29,15 @@ public void react(Slave context) throws Exception {
 		ConqueryConfig config = context.getConfig();
 		File dir = createWorkerName(context);
 		WorkerInformation info = new WorkerInformation();
-		info.setDataset(dataset.getId());
+		info.setDataset(dataset);
 		info.setIncludedBuckets(new IntArrayList());
 		info.setName(dir.getName());
-		WorkerStorage workerStorage = new WorkerStorageImpl(context.getValidator(), config.getStorage(), dir);
-		workerStorage.loadData();
-		workerStorage.updateDataset(dataset);
 
 
-		Worker worker = context.getWorkers().createWorker(info, workerStorage);
+		Worker worker = context.getWorkers().createWorker(info, config.getStorage(), createWorkerName(context), context.getValidator());
```

Comment:
`createWorkerName` wo ist die Methode definiert?

Reply:
Die ist in Workers definiert